File: Eulerpy/Euler9.py
#!/usr/bin/python
import math
i = 0
for a in range (1000):
	for b in range (1000):
		for c in range (1000):
			if a + b + c == 1000 and int(a**2 + b**2)**.5 == c and a and b and c:
				print a , b, c
				i = 1
				break
		if i: break
	if i: break

File: Eulerpl/Euler9.pl
#!/usr/bin/perl
use strict;
use warnings;
use 5.010;


for( my $a = 0; $a < 1000; $a++ ){
	for( my $b = 0; $b < 1000; $b++ ){
		for( my $c =0; $c < 1000; $c++ ){
			if($a**2 + $b**2 == $c**2 and 
				$a + $b + $c == 1000 and $a*$b*$c >0 ){
				say $a*$b*$c;
				exit;
			}
		}
	}
}












